test/e2e/lib/gce/gce.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import (
+	"net/http"
+	"time"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	compute "google.golang.org/api/compute/v1"
+)
+
+// GetComputeClient creates a GCE client with a 1 minute deadline.
+func GetComputeClient() (*compute.Service, error) {
+	const retries = 10
+	const backoff = time.Second * 6
```

Comment:
This was copied from https://github.com/kubernetes/kubernetes/blob/d1d6f64cdab34468a77c5e6773e778924f239be6/test/e2e_node/runner/remote/run_remote.go#L713

The code was last changed in 3 years ago. So I thought it would work, and didn't worry about it.

I'm happy to add some exponential backoff if you'd prefer.

Reply:
I see. Thanks for the explanation.